modules/uncertainty/R/get.parameter.samples.R

```diff
@@ -16,8 +16,8 @@ get.parameter.samples <- function(settings,
   pft.names <- list()
   outdirs   <- list()
   ## Open database connection
-  con <- PEcAn.DB::db.open(settings$database$bety)
-  on.exit(PEcAn.DB::db.close(con))
+  con <- try(PEcAn.DB::db.open(settings$database$bety), silent = TRUE)
+  on.exit(try(PEcAn.DB::db.close(con), silent = TRUE))
```

Comment:
See my comments above about (1) silencing errors, and (2) `on.exit(..., add = TRUE)`.